www/js/app/model/site/site_model.js

```diff
@@ -3,9 +3,7 @@ SiteModel = {
   sitePage: 0,
   create: function (attr, successCallback, errorCallback) {
     var cId = attr.collection_id;
-    var endpoint = getEndPoint();
-    var url = endpoint.URL_SITE + cId
-        + "/sites?auth_token=" + App.Session.getAuthToken();
+    var url = AppServerApi.getV1Collection() + cId + "/sites?auth_token=" + App.Session.getAuthToken();
```

Comment:
It's using in many places so it should be done by a method that return URL with auth_token parameter